xml/security_ldap.xml

```diff
@@ -889,7 +663,258 @@ telephoneNumber: +49 1234 567-10</screen>
+   for joining &ad; (which uses a combination of LDAP and Kerberos). For
+   more information, see <xref linkend="sec.security.auth.yast.client"/>.
+  </para>
+  <para>
+   Start the module by selecting
+   <menuchoice> <guimenu>Network Services</guimenu> <guimenu>LDAP and
+   Kerberos Client</guimenu> </menuchoice>.
+  </para>
+  <figure xml:id="fig.yast2.ldapkerberos.ldap">
+   <title><guimenu>LDAP and Kerberos Client</guimenu> Window</title>
+   <mediaobject>
+    <imageobject role="html">
+     <imagedata fileref="yast2_auth_client_config.png" width="65%" format="PNG"/>
+    </imageobject>
+    <imageobject role="fo">
+     <imagedata fileref="yast2_auth_client_config.png" width="65%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+  <para>
+   To configure an LDAP client, follow the procedure below:
+  </para>
+  <procedure xml:id="pro.security.auth.ldap">
+   <step>
+    <para>
+     In the window <guimenu>LDAP and Kerberos Client</guimenu>, click
+     <guimenu>Change Settings</guimenu>.
+    </para>
+    <para>
+     Make sure that the tab <guimenu>Use a Directory as Identity Provider
+     (LDAP)</guimenu> is chosen.
+    </para>
+    <informalfigure>
+     <mediaobject>
+      <imageobject role="html">
+       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
+      </imageobject>
+      <imageobject role="fo">
+       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
+      </imageobject>
+     </mediaobject>
+    </informalfigure>
+   </step>
+   <step>
+    <para>
+     Specify one or more LDAP server URLs, host names, or IP addresses under
+     <guimenu>Enter LDAP server locations</guimenu>. When specifying multiple
+     addresses, separate them with spaces.
+    </para>
+   </step>
+   <step>
+    <para>
+     Specify the appropriate LDAP distinguished name (DN) under
+     <guimenu>DN of Search Base</guimenu>. For example, a valid entry could
+     be <literal>dc=example,dc=com</literal>.
+    </para>
+   </step>
+   <step>
+    <para>
+     If your LDAP server supports TLS encryption, choose the appropriate
+     security option under <guimenu>Secure LDAP Connection</guimenu>.
+    </para>
+    <para>
+     To first ask the server whether it supports TLS encryption and be able
+     to downgrade to an unencrypted connection if it does not, use
+     <guimenu>Secure Communication via StartTLS</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Activate other options as necessary:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       You can <guimenu>Allow users to authenticate via LDAP</guimenu> and
+       <guimenu>Automatically Create Home Directories</guimenu> on the local
+       computer for them.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Use <guimenu>Cache LDAP Entries For Faster Response</guimenu> to cache
+       LDAP entries locally. However, this bears the danger that entries can
```

Comment:
I think this should be reworded to give an example of what the caching risk really is (and it depends on if they are using nscd or sssd).

Reply:
@Firstyear : Could you provide an example, please? I lack a bit of context here.

Reply:
Well, it depends what yast does .... anyway. If it's nscd, the cache risk is that when you define an authorisation rule (say, membersOf group admin) can login, and you "remove" a user from that group, the client cache will not see that change until the cache expires (or it refreshes). So there is a time window between the user remove and the affect on the server. The security risk here is that a revoked account can still use the login for a period of time later.

SSSD does not suffer this as it 'caches', but if online, it always checks the group memberships are still valid, so the reflection of the change is instant provided you are online. It's only if sssd is disconnected from the LDAP server that the cache risk exists.

Reply:
Thanks for the explanation! Added this information as a warning about a potential security risks with caching.